content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-wallet-server.md

```diff
@@ -0,0 +1,279 @@
+# Run rippled as a Wallet Server
+
+A Wallet Server is a catch-all configuration for rippled. With a Wallet Server, you can submit transactions to the XRP Ledger, access ledger history, and use the latest [tools](https://xpring.io/) to integrate with XRP.
```

Comment:
I think we should keep the link because xpring.io will change over time, but the overall theme is that it's where developers should go to for tools. The first subheading in xpring.io says "Xpring Platform provides developer tools, services, and programs to integrate money into your apps" so I think it's still appropriate. Although, I do see what you're saying about having to scroll down to actually get to the tools.